QuizApp/Quiz/Models/QuizResponseModel.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+struct QuizResponseModel: Decodable, Hashable {
+
+    let category: String
+    let description: String
+    let difficulty: String
```

Comment:
Difficulty has expected values, so it should be a enum. 
You need to create models for each layer, just like you did now with oteher models. 
e.g. DifficultyResponseModel, DifficultyDataSourceModel etc